scripts/globals/spells/damage_spell.lua

```diff
@@ -472,11 +472,6 @@ xi.spells.damage.calculateResist = function(caster, target, spell, skillType, sp
         magicAcc = magicAcc + 100 -- TODO: Confirm this value in retail
     end
 
-    -- Dark Seal
-    if casterJob == xi.job.DRK and skillType == xi.skill.DARK_MAGIC and caster:hasStatusEffect(xi.effect.DARK_SEAL) then
```

Comment:
Why was this removed? JP wiki seems to agree that nobody knows what it is, but they also agree that it's powerful like elemental seal. I can confirm Dark Seal more or less guarantees a land of an unresisted Drain 3 as long as its not dark resistant/immune. As long as it uses Elemental Seal value or less (there's data to show dispel is around +200 macc on it's own) I don't think it's all _that_ unreasonable

http://wiki.ffo.jp/html/3247.html

Reply:
As of the moment all dark magic is using the globals/magic.lua applyResistanceEffect function and never even hit this global. Should these be moved to use the xi.spells.damage.calculateResist function?

Reply:
They will, eventually, be moved to use a "not magic.lua" function. But that day is not today.

For now though, I would suggest leaving it there.

Reply:
Restored Dark Seal check to globals/spells/damage_spell.lua and added a check to globals/magic.lua.